go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 require (
 	cosmossdk.io/errors v1.0.0-beta.7
 	cosmossdk.io/math v1.0.0-beta.3
-	github.com/Gravity-Bridge/Gravity-Bridge/module v1.5.3
+	github.com/Gravity-Bridge/Gravity-Bridge/module v1.7.2
```

Comment:
Why changing this? It will be a non-compatible, breaking change. Please roll back.

Reply:
it doesn't change the code run.  But no worries, will change.

Reply:
up